io.github.Q1CHENL.fig.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "io.github.Q1CHENL.fig",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "fig",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=xdg-run/gvfsd",
+    "--filesystem=home"
```

Comment:
it uses gtk4, so it supports portals out of the box. Why does it need home access?